scripts/o3de/o3de/engine_template.py

```diff
@@ -2331,10 +2331,15 @@ def create_repo(repo_path: pathlib.Path,
         logger.error('Remote repository URI cannot be empty.')
         return 1
     
-    # check if a repo.json file already exist in directory - can only have one repo.json file per project
+    # check if path is empty and
+    if not force and repo_path.is_dir():
+        logger.error(f'{repo_path} already exists. Use the --force command to overwrite the existing directory.')
+        return 1
     repo_json_path = repo_path / 'repo.json'
-    if not force and repo_json_path.is_file():
-        logger.error(f'{repo_json_path} already exists.  Use the --force to overwrite the existing file.')
+    # if a repo.json file already exist in directory - can only have one repo.json file per project
+    if repo_json_path.is_file():
+        logger.error(f'{repo_json_path} already exists. If you want to edit the repo.json properties, use edit-repo-properties command.'
+                     'If you want to create a new remote repo template, create a new directory.')
```

Comment:
It's possible the user may want to re-create the file, and they shouldn't have to create a new directory to use this command.
```suggestion
        logger.error(f'{repo_json_path} already exists. If you want to re-create the existing file, use the --force option.  If you want to edit repo.json properties, use the edit-repo-properties command.')
```